tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/TlaFunOper.scala

```diff
@@ -82,15 +81,15 @@ object TlaFunOper {
     * We introduce a three-argument operator, whose arguments are as follows:</p>
     *
     * <ul>
-    * <li>function body of type TlaEx that may refer to the function via recFunRef,</li>
-    * <li>NameEx(variableName),</li>
-    * <li>variable domain of type TlaEx.</li>
+    *   <li>function body of type TlaEx that may refer to the function via recFunRef,</li>
+    *   <li>NameEx(variableName),</li>
+    *   <li>variable domain of type TlaEx.</li>
     * </ul>
     *
     * <p>Hence, a declaration of a recursive operator looks like a nullary operator declaration,
-    * whose body contains the constructor of a recursive function. The body of a recursive function may
-    * refer to the function itself by using the operator recFunRef (see below).
-    * Note that the output methods should convert this intermediate representation to the standard TLA+ form.</p>
+    *    whose body contains the constructor of a recursive function. The body of a recursive function may
```

Comment:
I wonder if it would be worth just running the formatter you're using on the entire code base and submitting that as a large (trivial) change? It would help reduce some noise from subsequent work.

Let's also put the formatting check CI? If you share the tool and config you're using for the formatting, I can make it happen :)

Reply:
Well, I am just using Intellij Idea. But it is slowly becoming a problem, as we have a mix of different code styles. Shall we switch to https://scalameta.org/scalafmt/ ?